mappings/UofM.py

```diff
@@ -25,6 +25,7 @@ def applyFormatting(self):
         self.record.has_part = []
         self.record.spatial = 'Michigan'
         self.record.source = 'UofM'
+        self.record.publisher_project_source = 'UofMichigan Press'
```

Comment:
Should this be UofMichigan Backlist to match the project name?

Reply:
The name is the name of the UofM table on Airtable so it's fine the way it is I believe.